slow_odgi/perturb.py

```diff
@@ -0,0 +1,18 @@
+import sys
+import mygfa
+import random
+
+def pop90percent(l):
+    origlen = len(l)
+    while len(l) > .1 * origlen:
+        l.pop(random.randrange(len(l)))
```

Comment:
FWIW, the [random](https://docs.python.org/3/library/random.html) module has something pretty similar to this:

```py
l[:] = random.sample(l, .1 * len(l))
```